README.md

```diff
@@ -12,33 +12,49 @@ features from the kit.
 
 The kit is composed of features, step definitions, and messages:
 
-- features, once executed, emit an exhaustive set of Messages as specified by
-  the protocol
-- step definitions allows to execute the features of the kit as expected
-- Messages - serialized as `.ndjson` files - are the reference: a given feature
+- **features**, once executed, emit an exhaustive set of Messages as specified by the protocol
+- **step definitions** that are used in conjunction with the features to generate the Messages, and also serve as reference step definitions for implementations adopting the kit
+- **Messages**, serialized as `.ndjson` files, are the reference: a given feature
   from the kit, executed using its dedicated step definitions, must emit the
   corresponding Messages
 
 The Messages from the kit are generated using
 [fake-cucumber](https://github.com/cucumber/fake-cucumber), which is used here
 as a reference implementation of the Messages protocol.
 
-## Contributing
+## Adoption
 
-This repo is composed of a [devkit](./devkit/), and packages to deliver the kit
-using managers like NPM or rubygem.
+To adopt the compatibility kit when writing a Cucumber implementation:
 
-### The devkit
+### Prerequisites
 
-The [devkit](./devkit/) is a set of tools to generate new Messages in order to
-extend the kit, or to update it after the protocol has been changed.
+First, make sure your implementation is sufficiently testable - you should be able to execute a test run programatically for a given set of feature files and step definitions, and get a handle on the Messages output (via a formatter or some other means) to make assertions on.
```

Comment:
and obtain the Messages output (in .ndjson format, via a formatter or some other means), to make assertions on.